server/reload_test.go

```diff
@@ -1571,6 +1571,9 @@ func TestConfigReloadClusterRemoveSolicitedRoutes(t *testing.T) {
 	if err := srvaConn.Flush(); err != nil {
 		t.Fatalf("Error flushing: %v", err)
 	}
+	if err := srvaConn.Flush(); err != nil {
```

Comment:
I am so sorry.. the flush was already done.. However, the test was failing when getting the reply back, so instead we should make sure that the subscription is correctly registered on serverB before sending the request.

Reply:
was wondering about 2 flushes. 
ok, will try that

Reply:
Replace what I asked you to add with this:
```
checkExpectedSubs(t, 1, srvb)
```

Reply:
... or, if you prefer, just revert the change here and we can handle the flappers in different PR..

Reply:
added the suggested check.